src/CasInterface.php

```diff
@@ -16,10 +16,13 @@ interface CasInterface
     /**
      * Authenticate the request.
      *
+     * @param array[] $parameters
+     *   The parameters related to the service. Can also contain a preset 'token'.
```

Comment:
Token is not a valid parameter key for the CAS protocol, I would then remove that sentence.